server/routers/advisor_router.py

```
@@ -425,3 +517,112 @@ async def post_chat(
     db.commit()
 
     return ChatResponse(reply=reply, conversationId=conv.id)
+
+
+@router.post("/chat/stream", summary="Send message and stream advisor reply")
+async def post_chat_stream(
+    payload: ChatRequestPayload,
+    db: Session = Depends(get_db_session),
```

Ruff (0.14.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)